src/lib/mongoBackend/MongoCommonRegister.cpp

```diff
@@ -82,7 +83,7 @@ static bool processSubscriptions
                                          trigs->attrL,
                                          mapSubId,
                                          trigs->reference,
-                                         trigs->format,
+                                         JSON,
```

Comment:
I am happy to include everything missing in the next PR.
We just need to keep track of the stuff to postpone/include.
Maybe a new issue ?